Silicon/ApollolakePkg/Library/FirmwareUpdateLib/FirmwareUpdateLib.c

```diff
@@ -495,18 +495,26 @@ PlatformEndFirmwareUpdate (
   return EFI_SUCCESS;
 }
 
+
 /**
-  Platform hook point to clear firmware update trigger.
+  Flash descriptor region lock
+
+  This function will do some command buffer parsing and check
+  for additional parameters
+
+  @param[in]  CmdDataBuf    Pointer to command buffer.
+  @param[in]  CmdDataSize   size of command data.
 
-  This function is responsible for clearing firmware update trigger.
+  @retval  EFI_SUCCESS      Flash descriptor lock successfully.
+  @retval  others           Error happening when updating.
 
 **/
-VOID
+EFI_STATUS
 EFIAPI
-ClearFwUpdateTrigger (
-  VOID
+SetFlashDescriptorLock (

```

Comment:
If it is a common interface, it should be
FwUpdateCmdHandler (Args)

Reply:
For APL, I have removed ClearFwUpdateTrigger () by mistake. I would fix it.